README.md

```diff
@@ -48,12 +48,12 @@ Note that helper contracts used by tests are held in the `v4-core/contracts/test
 ```markdown
 contracts/
 ----interfaces/
-    | IPoolManager.sol
-    | ...
+| IPoolManager.sol
+| ...
 ----libraries/
-    | Position.sol
-    | Pool.sol
-    | ...
+| Position.sol
```

Comment:
They were indented purposefully because they are inside the folder named! Please can you revert the file structure changes?